### PR TITLE
[Windows] change to reboot after Windows online updates installed

### PR DESCRIPTION
**Pull request by Tomorrow9**

### CODE REVIEW

**Review by keirazhang — DISMISSED** [review]
LGTM.

File: windows/utils/win_install_online_updates.yml [hunk 1]
```diff
@@ -54,6 +51,20 @@
   ansible.builtin.debug: var=win_updates_install_result
   when: enable_debug
 
+- name: "Restart guest OS when required"
+  when:
+    - not win_updates_install_result.failed | default(true)
```
Comment (123lzxm) [inline windows/utils/win_install_online_updates.yml:56]: It's better to reboot no matter win_updates_install_result.failed is true or false.

File: windows/utils/win_install_online_updates.yml [hunk 2]
```diff
@@ -8,7 +8,6 @@
 #     Some possible categories are: Application, Connectors, Critical Updates,
 #     Definition Updates, Developer Kits, Feature Packs, Guidance, Security Updates,
 #     Service Packs, Tools, Update Rollups, Updates and Upgrades.
-#     Default value is '*', which matches all categories.
```
Comment (123lzxm) [inline windows/utils/win_install_online_updates.yml:11]: Can keep the usage description for *. 
E.g. 
"Value * will match all categories."

File: windows/utils/win_install_online_updates.yml [hunk 3]
```diff
@@ -39,13 +38,11 @@
 - name: "Install Windows online updates"
   ansible.windows.win_updates:
     server_selection: "windows_update"
-    category_names: "{{ win_updates_categories | default('*') }}"
+    category_names: "{{ win_updates_categories | default(omit) }}"
```
Comment (123lzxm) [inline windows/utils/win_install_online_updates.yml:41]: Please help add category_names as input with value * in test case windows_update_install where the utility is called, to keep the behavior consistent. Thanks.

File: windows/utils/win_install_online_updates.yml (hunk 1 above)
Comment (Tomorrow9) [reply windows/utils/win_install_online_updates.yml]: Ok, changed. Thanks.

**Review by 123lzxm — APPROVED** [review]

**Pull request merged**